src/main/java/com/exasol/releaserobot/repository/maven/JavaMavenRepository.java

```diff
@@ -6,24 +6,28 @@
 import org.codehaus.plexus.util.xml.Xpp3Dom;
 import org.kohsuke.github.GHRepository;
 
-import com.exasol.releaserobot.repository.*;
+import com.exasol.releaserobot.repository.GitRepositoryException;
+import com.exasol.releaserobot.usecases.Repository;
 
 /**
  * This class represents a Maven-based Java project's content.
  */
-public class JavaMavenGitBranch extends AbstractGitHubGitBranch {
+public class JavaMavenRepository extends Repository {
```

Comment:
what is this a `JavaMavenRepository` and not just a `MavenRepository` ? is there something Java-related I'm missing?

Reply:
No,it's just a name. I renamed it